### ISSUE TITLE
Designating Pages for more regular spot checks

### ISSUE THREAD

**Issue by danielballan** [edgi-govdata-archiving/web-monitoring#114]
On the analyst call, it was noted that [one page](https://monitoring.envirodatagov.org/page/8ae30fcb-26aa-4fba-8c94-f5d097758ea7/..08fdef45-bf55-45df-86c2-0586599fc050) was dropping offline intermittently with unusual frequency, more than we usually see.

It might be useful to designate certain Pages for more regular spot checks (e.g. hourly). Maybe this is something to fit into the incipient work on a custom scraper.

**Comment by Mr0grog** [comment]
This should just be an extension of edgi-govdata-archiving/web-monitoring-processing#172. What’s the status with that, @weatherpattern? Are you planning to come back to it, or does it need someone else to take it over?

**Comment by Mr0grog** [comment]
Unless I’m misunderstanding what you’re getting at here, @danielballan.

**Comment by danielballan** [comment]
I was too vague here; let me try again.

Short of building a full scraper (or in addition to building a full scraper) we might run a service that regularly polls the response code for a set of important Pages. It wouldn't do anything about the content. The goal is to get better time resolution on the frequency and duration of outages than we can get from Versionista or IA or likely _any_ service that is pulling down and storing (or comparing) content.

**Comment by Mr0grog** [comment]
Aaaahhhhhhhhhhhhhhh, makes sense.

**Comment by Mr0grog** [comment]
Maybe treat it kinda like we do the IA healthcheck, and run it with cron and have it pick up a manifest of URLs to check from disk, then notify to Sentry if any return 400+ status codes (or don’t respond at all).
